components/mentions/style/patch.less

```diff
@@ -1,8 +1,18 @@
+@import '../../input/style/mixin';
+
 .ant-mention-dropdown {
-  top: 100%;
-  left: 0;
-  position: relative;
-  width: 100%;
-  margin-top: 4px;
-  margin-bottom: 4px;
+    top: 100%;
+    left: 0;
+    position: relative;
+    width: 100%;
+    margin-top: 4px;
+    margin-bottom: 4px;
+}
+
+.ant-mentions {
```

Comment:
I had to add the style change because the wrapping div was being styled previously, but when using the TextArea component that ends up having multiple boxes. I did it here instead of in the main styles less because I am under the impression that those get overwritten by the style sync jobs.